bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/BusyIndicator.java

```diff
@@ -70,6 +70,36 @@ public static void showWhile(Display display, Runnable runnable) {
 		}
 	}
 
+	/**
+	 * If called from a {@link Display} thread waits for the given
+	 * <code>Future</code> to complete providing busy feedback using the
```

Comment:
It feels like there should be a coma after "complete" otherwise the phrase just reads on.